ui/src/index.tsx

```diff
@@ -19,7 +18,7 @@ import WebSocketClient from "./websocket-client";
 const { createReduxHistory, routerMiddleware, routerReducer } =
   createReduxHistoryContext({
     history: createBrowserHistory({
-      basename: generateNewURL(),
+      basename: `${process.env.REACT_APP_BASENAME}${process.env.REACT_APP_REACT_BASENAME}`,
```

Comment:
Once the migration is complete this will just become `process.env.REACT_APP_BASENAME`